mods/ctf_pvp_engine/ctf_chat/init.lua

```diff
@@ -322,8 +394,9 @@ minetest.register_chatcommand("teamleave", {
 			end
 			if disband == true then
 				if ctf.remove_team(team) then
-					ctf.needs_save = true
+					ctf.access_remove_team_all(team)
```

Comment:
If `ctf.access_remove_team_all` has to be called every time `ctf.remove_team` is called, then just combine the former into the latter. i.e. `ctf.remove_team` should automatically take care of removing the team from all access lists.